### DISCUSSION TITLE
Where to translate Play / Pause buttons on the map

### DISCUSSION THREAD

**Discussion by christoph-LSN** [open-sdg/open-sdg#1821] [Q&A] [answered]
@MoBosse 
Where can I translate the "Play", "Pause", "Backward" and "Forward" buttons on the map timeslider?

![image](https://user-images.githubusercontent.com/94443931/193781572-a7252b0d-72ed-4de3-befc-17455c84f9f0.png)

**Comment by MoBosse** [comment, marked as answer]
@christoph-LSN I think this is hard coded [here](https://github.com/socib/Leaflet.TimeDimension/blob/master/src/leaflet.timedimension.control.js#L161). Looks to me like it's not something that can be easily customized.

**Comment by christoph-LSN** [comment]
@MoBosse Ok, then we have to live with Backward and Forward.....

**Comment by brockfanning** [comment]
@MoBosse @christoph-LSN For your awareness, in 2.1.0 the play/pause button has been removed. The titles for the other buttons (backwards, forwards, etc.) will be translated in 2.1.0 as well. We hope to release 2.1.0 in the very near future.
